packages/block-editor/src/components/media-placeholder/styles.native.scss

```diff
@@ -1,50 +1,63 @@
-.emptyStateContainer {
+.media-placeholder__container {
 	flex: 1;
 	height: 142;
 	flex-direction: column;
 	align-items: center;
 	justify-content: center;
-	background-color: $gray-lighten-30;
+	background-color: #e0e0e0; // $light-dim
```

Comment:
Similar to my reply above, the issue is that `$light-dim` has an alpha value, so depending on the background it would show differently, so for this case, since it's the placeholder's container, we are using a solid color that's similar to `$light-dim` having it on top of a white background color.